### PR TITLE
Fixed data-insert to work with new stub

### PR DESCRIPTION
**Pull request by matt712**

### CODE REVIEW

File: insert_data.sh [hunk 1]
```diff
@@ -34,3 +34,47 @@ curl -X POST -H "Content-Type: application/json" \
 	}
 }' \
 http://localhost:9303/setup/data
+
+curl -X POST -H "Content-Type: application/json" \
```
Comment (robert-webb-hmrc) [inline insert_data.sh:38]: Do we need this request?

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]

File: insert_data.sh (hunk 1 above)
Comment (matt712) [reply insert_data.sh:38]: I'm not sure, it depends on how the api handles it.

Comment (robert-webb-hmrc) [reply insert_data.sh:38]: Less of a question about the other posts (Although removing all the post requests does allow the tests to run) more about the one post request to 1234567890 as we are not using this mtditd in any scenarios

Comment (matt712) [reply insert_data.sh:38]: Yeah just tested it, we don't need it

**Review by robert-webb-hmrc — APPROVED** [review]

**Pull request merged**